src/QueueReceiver.Infrastructure/EntityConfigurations/PlantConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public void Configure(EntityTypeBuilder<Plant> builder)
             builder.Property(plant => plant.PlantId).HasColumnName("PROJECTSCHEMA");
             builder.Property(plant => plant.AffiliateGroupId).HasColumnName("AFFILIATEGROUPID");
             builder.Property(plant => plant.InternalGroupId).HasColumnName("INTERNALGROUPID");
+            builder.Property(plant => plant.IsVoided).HasColumnName("ISVOIDED");
```

Comment:
We usually map the ISVOIDED flag (found in various tables) to bool types. This can be done in the mapping (ref `PersonConfiguration`):

            builder.Property(p => p.IsVoided).HasColumnName("ISVOIDED")
                .HasConversion(
                    b => b ? 'Y' : 'N',
                    c => c.Equals('Y'));

Reply:
Done